numerapi/numerapi.py

```diff
@@ -1,12 +1,13 @@
 # -*- coding: utf-8 -*-
 
+from __future__ import absolute_import
```

Comment:
I've removed python2 support recently and removed that future import. Any reason for re-introducing it or did you just branch of before the removal?

Reply:
not intended